frontend/src/components/auth/StudentLogin/StudentNameSelection.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import {
+  FormControl,
+  FormLabel,
+  Button,
+  FormErrorMessage,
+} from "@chakra-ui/react";
+import { Select } from "chakra-react-select";
+
+import { Controller, useForm } from "react-hook-form";
+import { useHistory } from "react-router-dom";
+import AuthWrapper from "../AuthWrapper";
+import { STUDENT_SIGNUP_IMAGE } from "../../../assets/images";
+import { LeftArrowIcon } from "../../../assets/icons";
+import * as Routes from "../../../constants/Routes";
+
+const students = [
+  {
+    label: "Evans",
+    value: "Evans",
+  },
+  {
+    label: "Tony",
+    value: "Tony",
+  },
+  {
+    label: "John",
+    value: "John",
+  },
+  {
+    label: "Enzo",
+    value: "Enzo",
+  },
+  {
+    label: "Julia",
+    value: "Julia",
+  },
+  {
+    label: "Emily",
+    value: "Emily",
+  },
+  {
+    label: "Afsana",
+    value: "Afsana",
+  },
+  {
+    label: "George",
+    value: "George",
+  },
+  {
+    label: "Cyrus",
+    value: "Cyrus",
+  },
+];
+const StudentNameSelection = (): React.ReactElement => {
+  const history = useHistory();
+  const { control } = useForm();
+  const title = "Student Login";
+  const subtitle = "Please enter your name in the field or search for it below";
+  const image = STUDENT_SIGNUP_IMAGE;
+  const form = (
+    <Controller
+      control={control}
+      name="grade"
+      rules={{ required: "Please select a grade" }}
+      render={({ field: { onChange, value, name }, fieldState: { error } }) => (
+        <FormControl isRequired isInvalid={Boolean(error)}>
+          <FormLabel color="grey.400">First Name</FormLabel>
+          <Select
+            name={name}
+            onChange={onChange}
+            options={students}
+            value={value}
+            placeholder="Search Name by typing it in field"
+            useBasicStyles
+          />
+          <Button variant="primary" mt="10%" width="100%">
+            Continue
+          </Button>
+          <Button
+            onClick={() => {
+              history.push(Routes.HOME_PAGE);
+            }}
+            mt="10%"
+            width="100%"
+            leftIcon={<LeftArrowIcon />}
+            textStyle="paragraph"
+            fontWeight="500"
+            variant="link"
+            _hover={{ color: "blue.200", textDecoration: "none" }}
+            color="blue.300"
+          >
+            Back to Home
+          </Button>
```

Comment:
Instead of defining the buttons, are you able to use existing code for [Navigation Buttons](https://github.com/uwblueprint/jump-math/blob/staging/frontend/src/components/auth/TeacherSignup/NavigationButtons.tsx#L24)?